packages/core/src/zone/ng_zone.ts

```diff
@@ -119,7 +119,11 @@ export class NgZone {
   readonly onError: EventEmitter<any> = new EventEmitter(false);
 
 
-  constructor({enableLongStackTrace = false, shouldCoalesceEventChangeDetection = false}) {
+  constructor({
+    enableLongStackTrace = false,
+    shouldCoalesceEventChangeDetection = false,
+    shouldCoalesceChangeDetection = false
```

Comment:
nit: make naming consistent

```suggestion
    shouldCoalesceRunChangeDetection = false
```

Reply:
Got it.